Resources/views/userEmailsDashboard.html.twig

```diff
@@ -0,0 +1,93 @@
+{% extends 'AzineEmailBundle::base.html.twig' %}
```

Comment:
dry: this whole file is just a copy-paste from `emailsDashboard.html.twig`. Right?!